policies-procedures-and-docs/cohort-notes/q-and-a-call.md

```diff
@@ -0,0 +1,132 @@
+
```

Comment:
That could be   added to the latest cohort repo instead

Reply:
Moving into [OLS-8](https://github.com/open-life-science/ols-8)...

Reply:
Please, see PR [#74](https://github.com/open-life-science/ols-8/pull/74).